src/js/models/v2/awards/financialSystemDetails/BaseFinancialSystemDetailsRow.js

```diff
@@ -12,8 +12,10 @@ export const formatDate = (date) => date.format('MM/DD/YYYY');
 const BaseFinancialSystemDetailsRow = {
     populate(data) {
         this.id = data.financial_accounts_by_awards_id || '';
+        this.reporting_fiscal_year = data.submission.reporting_fiscal_year || null;
```

Comment:
Could you use came case variables to match the naming convention used in our models? I would also prefix both year and quarter with an underscore to indicate that they are not formatted/ only accessed by the getter.